protocol/protos/WifiCore.proto

```diff
@@ -89,3 +89,13 @@ message AccessPointConfiguration
 {
     Dot11PhyType PhyType = 1;
 }
+
+message AccessPointCapabilities
+{
+    repeated Microsoft.Net.Wifi.RadioBand SupportedBands = 1;
+}
+
+enum AccessPointState

```

Comment:
Is this used anywhere? Maybe should be part of `AccessPointDescriptor`?

Reply:
It's not used anywhere yet. I initially thought it would go in `AccessPointDescriptor`, but that is intended to describe the static, unchanging properties of an access point. So, if you think that makes sense, then I think the question would be should `AccessPointState` be included in the `WifiEnumerateAccessPointsResult` for each access point? For example, instead of providing a list of `AccessPointDescriptor`s, we'd instead do something like:

```proto
message WifiEnumerateAccessPointsRequestResultEntry
{
    AccessPointDescriptor Descriptor;
    AccessPointState State;
}

message WifiEnumerateAccessPointsResult
{
    repeated WifiEnumerateAccessPointsRequestResultEntry AccessPoints = 1;
}
```

We would need to query each enabled AP to get the state. That makes this AP more active, and the idea was to make it something the clients could use simply to see what APs were available and their capabilities. So, my initial thought is not to include it and instead, have a separate API that allows interaction with the AP to do active queries/actions with it, like querying its current state, changing certain settings, etc.

Thoughts?

Reply:
Oh I see, I think it makes sense to leave `AccessPointDescriptor` as something that contains the unchanging properties, although could `IsEnabled` be changed?

I think your initial idea to have a separate API to query info about the AP like its current state is a good idea, so no need to change anything here

Reply:
Great point about `IsEnabled`, it shouldn't be in there. I'll factor it out.

Reply:
I ended up removing `AccessPointDescriptor` all together because it didn't make sense to exist on its own. To that end, I moved all of the result arguments into their own message `WifiEnumerateAccessPointResultItem`.